package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -205,9 +207,15 @@ export const renderText = <
   const match: MatchFunction = (source) => regEx.exec(source);
 
   const mentionsReact: ReactNodeOutput = (node, output, { ...state }) => {
+    const userName = node.content[0]?.content.replace(/[|&;$%@"<>()+,]/g, '');
```

Comment:
It would be nice if you can add tests for this or atleast a comment about what this does

Reply:
Ohhh... this is a regex to remove the `@` sign from the userName string. I've added documentation